leetcode/BinaryTreeTests/BinaryTree_545.cs

```diff
@@ -0,0 +1,81 @@
+﻿namespace BinaryTreeTests;
+
+[TestFixture]
+internal class BinaryTree_545
+{
+    class Solution {
+        private enum NodeType
+        {
+            Root,
+            LeftBound,
+            RightBound,
+            Other
+        }
+
+        public IList<int> BoundaryOfBinaryTree(TreeNode root) {
```

Comment:
1. Need to identify child's type base on parent's type
2. Left child and right child have different logic